src/button/cta.scss

```diff
@@ -2,33 +2,41 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import './button';
+@import './disabled';
 
 @mixin iui-button-cta {
-  @include themed {
-    background-color: t(iui-color-background-positive);
-    border-color: t(iui-color-background-positive);
-    color: t(iui-color-foreground-accessory);
-  }
-
-  > .iui-icon {
+  &,
+  &:focus:not(:focus-visible) {
     @include themed {
-      fill: t(iui-color-foreground-accessory);
+      background-color: t(iui-color-background-positive);
+      border-color: t(iui-color-background-positive);
+      color: t(iui-color-foreground-accessory);
+    }
+
+    > .iui-button-icon {
+      @include themed {
+        fill: t(iui-color-foreground-accessory);
+      }
     }
   }
 
-  &:enabled:hover,
-  &:enabled:active,
-  &:focus-visible {
+  &:hover,
+  &:active,
+  &:focus {
     @include themed {
       background-color: t(iui-color-background-positive-overlay);
       border-color: t(iui-color-background-positive-overlay);
       color: t(iui-color-foreground-accessory);
     }
 
-    > .iui-icon {
+    > .iui-button-icon {
       @include themed {
         fill: t(iui-color-foreground-accessory);
       }
     }
   }
+
+  &[disabled] {
```

Comment:
Can we add a class name here in addition to this?  Something like `&[disabled], &.iui-disabled`.

Reason I'm asking is when a button is disabled, it can no longer show a tooltip on hover.  I think if we offer a class name that makes a button appear disabled, we should be able to show a tooltip on hover / focus.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/849817/140779502-d6f28363-2acc-464f-9857-58a38fbb895b.png)

Reply:
That's a bad idea.

Disabled buttons cannot be interacted with in any way (i.e. no click, no focus, AND **no focus** which is why https://github.com/iTwin/iTwinUI-react/issues/30 is a known bug). If we were to use a class name instead of the `disabled` attribute, it would require manually disabling the click handler and removing the button from tabbing sequence and adding aria labels for screen readers, and who knows what else.

The workaround for tooltips is to add a `div` around the disabled button, but that also breaks our button-group styling.

Reply:
Here's an article I bookmarked a while back that should help here: https://css-tricks.com/making-disabled-buttons-more-inclusive/

Reply:
Yeah so it looks like the discussion should be centered around these 5 points from the article. And we can narrow it down to the top 3 points (because semantics and styles are covered with CSS and aria-disabled):
![image](https://user-images.githubusercontent.com/9084735/141002272-537dc047-7483-4862-8f5b-5886df9cee5f.png)

If we choose to make disabled buttons focusable and hoverable, then it will introduce an inconsistency with other disabled elements, such as tabs and menu items. So this decision needs to be very carefully finalized.

Reply:
So should I add `.iui-disabled` to disabled styles?

Reply:
Lets save this for another PR. Created issue #391